docs/user-guide/aroma/installing-payloadloader.md

```diff
@@ -0,0 +1,26 @@
+# Aroma {docsify-ignore-all}
+
+## Installing PayloadLoader
+
+Now that you have your NAND backup in case anything goes wrong later, you can install the PayloadLoader on your system.
+
+Installing the PayloadLoader will let you access Aroma just by running the Health and Safety Information app.
+
+!> A factory reset **won't** uninstall the injected PayloadLoader. Follow [uninstall PayloadLoader](../../uninstall-payloadloader) to remove the PayloadLoader. 
+
+### Instructions
+
+?> If you're already booted into Aroma, you may skip to step 5.
+
+1. Take the SD Card out of your PC and plug it into your Wii U.
+1. Execute the web browser exploit as [previously explained](browser-exploit) but this time, you will need to hold the X button to open the Environment Loader menu.
+1. Press A to launch the `aroma` environment.
+1. Press A to launch the Wii U Menu.
+1. At the Wii U Menu, launch the PayloadLoader Installer.
```

Comment:
It probably makes sense to explain a bit more here. A lot of people won't be familiar with the concept of having new titles appear on the Wii U Menu.
Something like:
```suggestion
1. At the Wii U Menu, look for the newly appeared `PayloadLoader Installer` title (the round circle with an A inside of it) and launch it.
```